### DISCUSSION TITLE
Small suggestion to Song Selection User Interface

### DISCUSSION THREAD

**Discussion by RoyceThomas** [ppy/osu#16878] [Ideas] [answered]
hello. I had once saw in [osu! lazer 2022.205](https://osu.ppy.sh/home/changelog/lazer/2022.205.0), the bottom toolbar at song select where mods, random, and options in my opinion looked so much better with the decreased opacity (as compared to the old and current one (in the attachment below). idk if this was a glitch but basically the bar that I click on in the video was more transparent and still blurred the area behind it. 

i believe that this might look better and more modern and feel free to change it back if people don't like but pls try it out at least. the colored line when hovering/selecting the buttons on it (mods, random, and options) still appeared tho.

https://user-images.githubusercontent.com/54102527/154037961-4da4d196-b8a7-41b2-a645-33ded49ff6ee.mp4

**Comment by peppy** [comment, marked as answer]
The whole design is changing, and will likely become more opaque again.

**Reply by RoyceThomas** [reply]
awesome! thanks for the heads up peppy!
